docs/configuration.rst

```diff
@@ -49,6 +49,15 @@ Then add the ``admin_tools.template_loaders.Loader`` template loader to your
     begining of the list in your ``TEMPLATES`` or ``TEMPLATE_LOADERS``
     settings variable.
 
+.. important::
+    it is very important that ``django.contrib.staticfiles.finders.AppDirectoriesFinder''
+    be there in your ``STATICFILES_FINDERS``.
+
+.. important::
+    Starting from django 2.1, using ``template_loaders`` option require making disable 
+    ``APP_DIRS`` option and you have to put the ``admin_tools.template_loaders.Loader`` together with
+    ``admin_tools.template_loaders.Loader``
```

Comment:
There is some language issue here. `admin_tools.template_loaders.Loader` is mentioned twice.

@hdformat Can you try to explain in more words what you want to express?

Reply:
@hdformat can you please fix the english, I'm not native but "making disable" is not correct I think, maybe something like:
"Starting from django 2.1, you must set ``APP_DIRS`` to ``False`` when using ``templates_loaders``..."

Also, you repeat twice the same loader in the last sentence.

Please update the PR and I'll merge it ASAP.